xesmf/frontend.py

```diff
@@ -271,6 +272,14 @@ def __init__(
             If not given or if those are not found on the regridded object, regridding
             uses the two last dimensions of the object (or the last one for input LocStreams and Meshes).
 
+        default_weights: boolean, optional
```

Comment:
I think the `default_weights` is too vague, let's find a name that describe what the arg is doing.
would `unmapped_to_nan` work?

Reply:
`unmapped_to_nan` seems to be an appropriate name. I will adjust it.